### PR TITLE
l10n: Spelling unification

### PR DESCRIPTION
**Pull request by Valdnet**
Spelling unification in Transifex.

Signed-off-by: Valdnet <47037905+Valdnet@users.noreply.github.com>

### CODE REVIEW

File: src/components/TimelineList.vue [hunk 1]
```diff
@@ -96,7 +96,7 @@ export default {
 				notifications: {
 					image: 'img/undraw/notifications.svg',
 					title: t('social', 'No notifications found'),
-					description: t('social', 'You haven\'t receive any notifications yet')
+					description: t('social', 'You have not receive any notifications yet')
```
Comment (rakekniven) [inline src/components/TimelineList.vue:99]: receive -> received

**Review by rakekniven — CHANGES_REQUESTED** [review]
Info: This app is not supported anymore. No release for NC22.

**Review by rakekniven — APPROVED** [review]

**Comment by Valdnet** [comment]
> Info: This app is not supported anymore. No release for NC22.

So I can close this PR if the application is no longer supported?

**Comment by rakekniven** [comment]
@Valdnet are you reading at https://cloud.nextcloud.com/call/xs25tz5y ?
I asked about social and the app is not dead nor alive 🤔

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**